label_studio_ml/tesseract/tesseract.py

```diff
@@ -82,15 +87,20 @@ def predict(self, tasks, **kwargs):
             w = meta["width"] * meta["original_width"] / 100
             h = meta["height"] * meta["original_height"] / 100
 
-            result_text = pt.image_to_string(image.crop((x, y, x + w, y + h)), config=OCR_config).strip()
+            page = document[page_number]
+            result_text = fitz.utils.get_text(
+                page, "words", clip=[x / 3, y / 3, (x + w) / 3, (y + h) / 3]
+            )  # we use a magnifier of three, that's why we have to divide by three.
```

Comment:
Can we rescale the coordinates to have values between 0 and 1 (for both x and y), before they are sent here?
Then they would be independent of the any rescaling or switching between different units (PDF points vs. pixels).

Reply:
Do you mean if we can rescale them at the beginning of the `predict`  method?